pages/about.html

```diff
@@ -4,9 +4,43 @@
   <meta charset="UTF-8">
   <meta http-equiv="X-UA-Compatible" content="IE=edge">
   <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <title>Document</title>
+  <title>✨About Me✨</title>
+  <link rel="stylesheet" href="../styles/style.css">
+  
 </head>
+<header>
+  <h1>K. Buslach </h1>
+  <h1>ADA C17 Sharks 🦈</h1>
+</header>
+<div class="main">
+<div class="navigate">
+  <ul>
+  <nav>
+    <li><a href="../index.html">Home</a></li>
+    <li><a href="/about.html">About me</a></li>
+    <li><a href="/portfolio.html">Portfolio</a></li>
+  </nav>
+  </ul>
+</div>
+
+  <div class="pictures">
+    <img class="winnie" src="../images/Winnie.JPG" alt="Picture of a small dog sniffing the air">
+    <img class="draw" src="../images/malibu.png" alt="Drawing of a woman on a pier">
+    <img class="winnie" src="../images/IMG_9979.PNG" alt="Picture of a  small dog with flowers in her hair">
+  </div> 
+</div>
+
+<h2> A little about me:</h2>
```

Comment:
This should all live inside the body tag on line 34 along with the your header element above since body element represents the content of an HTML document.